medical_physician/migrations/10.0.2.0.0/post-migrate.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, SUPERUSER_ID
+from odoo.tools import mute_logger
+
+
+# Mute logger in physician model to prevent write error messages
+@mute_logger('odoo.addons.medical_physician.models.medical_physician')
+def migrate(cr, version):
+    """Create practitioner record for each physician and clean up"""
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    physicians = env['medical.physician'].search([])
+
+    # Use new specialty since old record will be removed
+    old_gp_specialty = env.ref('medical_physician.medical_specialty_gp')
+    gp_specialty = env.ref('medical_practitioner.medical_specialty_gp')
```

Comment:
For uniformity I think this should be `new_gp_specialty`.

But for double awesomeness, the names should be reversed a bit. When naming grouped/similar objects, think high level then drill to granular. In this  example:

```
specialty_gp_new
specialty_gp_old
```